components/compliance-service/api/reporting/server/server.go

```diff
@@ -914,3 +915,40 @@ func (srv *Server) GetReportContent(ctx context.Context, in *reporting.ReportCon
 	}
 	return nil*/
 }
+func (srv *Server)AssetCount(ctx context.Context , in *reporting.ListFilters) (*reporting.AssetSummary , error) {
+	formatttedFilters := formatFilters(in.Filters)
+	var count reporting.AssetSummary
+	if len(formatttedFilters["start_time"]) <= 0 || len(formatttedFilters["end_time"]) <= 0 {
+		return nil , &errorutils.InvalidError{}
+	} else if reflect.DeepEqual(formatttedFilters["start_time"] , formatttedFilters["end_time"]) {
+		return nil , &errorutils.InvalidError{}
+	} else {
+		start_date := formatttedFilters["start_time"]
```

Comment:
validate this in another function. Rename the variables as well. Create a function which validated start time and end time. Take the layout as ttime.RFC3339.